lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers_test.go

```diff
@@ -434,6 +434,32 @@ func TestListSubscribers(t *testing.T) {
 		ExpectedResult: tests.JSONMarshaler(expectedResult),
 	}
 	tests.RunUnitTest(t, e, tc)
+
+	expectedResultAbbreviated := subscriberModels.PaginatedSubscribersAbbreviated{
+		TotalCount:    expectedResult.TotalCount,
+		NextPageToken: expectedResult.NextPageToken,
+	}
+
+	for k := range expectedResult.Subscribers {
+		expectedResultAbbreviated.Subscribers = append(expectedResultAbbreviated.Subscribers, k)
+	}
+
+	if len(expectedResult.Subscribers) == 0 || expectedResult.TotalCount == 0 {
+		t.Errorf("Checking output schema of empty subscriber list? That's a problem!" +
+			"Make sure there's at least one subscriber.")
```

Comment:
Can't tell how long this line is, but generally don't break unless it's 120 chars or even a bit more